draft-duke-quic-load-balancers.md

```diff
@@ -560,6 +560,16 @@ configuration at that codepoint have closed or transitioned.  They MAY drop
 connection IDs with the old configuration after a reasonable interval to
 accelerate this process.
 
+## Configuration Failover
+
+If a server is configured to expect QUIC-LB messages, and it has not received
+these, it MUST generate connection IDs with the config rotation bits set to
+'0b11' and MUST use the "disable_migration" transport parameter in all new
```

Comment:
I'm not sure we should assume anything about a presence of a load balancer at all, if no configuration came down. I think the server should be able to do whatever it wants with the CIDs then and not need to set any particular bits.

Reply:
A server must at least have been configured with a token, which means "expect QUIC-LB packets". If we don't get any, there's either a misconfiguration or a middlebox is eating QUIC-LB. In either case I think it's wise to revert to 5-tuple routing.